materials/custom-properties.css

```diff
@@ -150,7 +155,8 @@
   --font-family-default: 'Open Sans', sans-serif;
   --font-size-default: 13px;
   --font-size-small: 12px;
-  --shadow-box-tag-hover: 0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
+  --shadow-box-tag-hover: 0 1px 3px rgba(0, 0, 0, 0.12),
+    0 1px 2px rgba(0, 0, 0, 0.24);
```

Comment:
This only changed as we run prettier on them now, correct?

Reply:
Yes correct